src/engine/client/cl_cgame.cpp

```diff
@@ -1485,35 +1484,33 @@ void CGameVM::QVMSyscall(int index, Util::Reader& reader, IPC::Channel& channel)
 
 		case CG_KEY_GETKEYNUMFORBINDS:
 			IPC::HandleMsg<Key::GetKeynumForBindsMsg>(channel, std::move(reader), [this] (int team, const std::vector<std::string>& binds, std::vector<std::vector<int>>& result) {
-                for (const auto& bind : binds) {
-                    result.push_back({});
-                    for (int i = 0; i < Util::ordinal(keyNum_t::MAX_KEYS); i++) {
-                        char buffer[MAX_STRING_CHARS];
-
-                        Key_GetBindingBuf(i, team, buffer, MAX_STRING_CHARS);
-                        if (bind == buffer) {
-                            result.back().push_back(i);
-                            continue;
-                        }
-                        Key_GetBindingBuf(0, team, buffer, MAX_STRING_CHARS);
-                        if (bind == buffer) {
-                            result.back().push_back(i);
-                            continue;
-                        }
-                    }
-                }
+				for (const auto& bind : binds) {
+					result.push_back({});
+					for (int i = 1; i < Util::ordinal(keyNum_t::MAX_KEYS); i++) {
+						if (i >= 'A' && i <= 'Z') {
```

Comment:
Could you add a comment on why you can skip these? Are we only allowed to return binds for keyboard keys here?

Reply:
If it looks up both 'a' and 'A' then those binds are double-counted. The whole thing is a temporary implementation; to really work the API needs to change.

Reply:
Sounds like a good comment to include in the source code.

Reply:
Done.